### ISSUE TITLE
We couldn't make changes to your system. Please install or uninstall 'ipfs-npm' package manually

### ISSUE THREAD

**Issue by ProtiJay** [ipfs-shipyard/ipfs-webui#1165]
We couldn't make changes to your system. Please install or uninstall 'ipfs-npm' package manually

**Comment by hacdias** [comment]
Hello @ProtiJay! It would be quite nice to have a bit more information. Did that message appear out of nowhere or after you ticked the `npm on ipfs` box?

**Comment by Kuurusch** [comment]
I've got this issue as well. It appears, when I open in the IPFS-Desktop application the "status".

**Comment by hacdias** [comment]
@Kuurusch did you have `ipfs-npm` installed before? I'm trying to figure out the reason why we're displaying this message when it shouldn't...

**Comment by ProtiJay** [comment]
I uninstalled and reinstalled and the issue did not pop up again.

**Comment by Kuurusch** [comment]
@hacdias No, not that I know... I searched on my machine and didn't found any! I'm using a Mac.

**Comment by hacdias** [comment]
@Kuurusch did it happen just once or every time you open IPFS Desktop?

**Comment by KurtPfeifle** [comment]
I see the same message when I open IPFS Desktop settings on a Mac ("Sierra"). It appears as a sort of popup window at the bottom of the screen.

When I scroll down, I see the checkmark for "Enable ipfs-nmp" being disabled. (This is a fresh first install, and I ***never*** had it enabled even.)

**Comment by KurtPfeifle** [comment]
Poking around the system a bit, I found this [error.log](https://github.com/ipfs-shipyard/ipfs-webui/files/3719690/error.log) located at *"Library/Application\ Support/IPFS\ Desktop/"*.

Already the first line says:

     [npm on ipfs] Error: Command failed: npm install -g ipfs-npm

So this thingie tries to run a command line behind my back without informing me???

**Comment by hacdias** [comment]
Hello @KurtPfeifle! Definitely there is something wrong here as it shouldn't. I will take a look and get this sorted out in a day or two!
